tox.ini

```diff
@@ -1,80 +1,79 @@
 [tox]
 envlist = flake8, pylint, check-{py27,py36}, coveralls
-skipsdist = true
-skip_missing_interpreters = True
 
 [testenv]
 basepython = python2.7
 
-[base]
-passenv = *
-setenv =
-    PYTHONPATH = {toxinidir}/src
-    LC_ALL = C
+[testenv:check-py27]
 deps =
+    -rrequirements.txt
     pytest-cov
     pytest==3.5.1
-changedir = {toxinidir}/tests
-
-[testenv:check-py27]
-passenv = {[base]passenv}
-setenv =
-    {[base]setenv}
-deps = {[base]deps}
-changedir = {[base]changedir}
 basepython = python2.7
+changedir = {toxinidir}/tests
 commands =
     pytest \
         --durations=5 \
         --cov=libnmstate \
+        --cov=nmstatectl \
         --cov-report=html:htmlcov-py27 \
         {posargs} \
-        lib
+        libnmstate \
+        nmstatectl
 
 [testenv:check-py36]
-passenv = {[base]passenv}
-setenv =
-    {[base]setenv}
-deps = {[base]deps}
-changedir = {[base]changedir}
+deps =
+    -rrequirements.txt
+    pytest-cov
+    pytest==3.5.1
 basepython = python3.6
+changedir = {toxinidir}/tests
 commands =
     pytest \
         --durations=5 \
         --cov=libnmstate \
+        --cov=nmstatectl \
         --cov-report=html:htmlcov-py36 \
         {posargs} \
-        lib 
+        libnmstate \
+        nmstatectl
 
 [testenv:pylint]
-setenv =
-    {[base]setenv}
+skip_install = true
 deps =
     pylint==1.8.4
 commands =
     pylint \
         --errors-only \
         {posargs} \
-        libnmstate 
+        libnmstate \
+        nmstatectl \
+        tests/libnmstate/helloworld_test.py \
```

Comment:
Mentioning each module is not maintainable, why is it needed?

Reply:
Pylint is not able to gather test modules since they are not in a package (missing __init__.py), there is opened RFE for it https://github.com/PyCQA/pylint/issues/352

Reply:
It should be enough to mention the package (like before: tests/libnmstate).

Reply:
The packages under tests should have __init__.py files.